src/bse202/__init__.py

```diff
@@ -1,29 +1,35 @@
 from .db import check_db_exists, get_db, delete_db
-
 from .hooks import register_hooks
-
 from .routes import root_blueprint
 from .routes.auth import auth_blueprint
 from .routes.store import store_blueprint
 from .routes.user import user_blueprint
 
 from flask import Flask, g
 from os import environ
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv(".env")
 
 app = Flask(__name__)
 
+# Set secret key from environment variable
+app.config["SECRET_KEY"] = environ.get("SECRET_KEY", "default_secret_key")
```

Comment:
unnecessary, handled in [hooks](https://github.com/d3rpp-dev/bse203/tree/main/src/bse202/hooks.py)